docs/faq.md

```diff
@@ -0,0 +1,33 @@
+# General FAQ
+
+## AMD GPUs crashing the entire system
+
+Changing resolution, using DRI_PRIME and doing various other things can cause crashes, but running the command `echo high| sudo tee /sys/bus/pci/drivers/amdgpu/0000:??:??.?/power_dpm_force_performance_level` or adding the `amdgpu.dpm=0` to the kernel commandline stops these crashes.
+
+## The MacBook's screen freezes, fans run up, and then it turns off
+
+A broken runtime call crashed Linux; try a patched kernel (or updating if you're already using it), or boot macOS and check the crashlog. If you have `apple-bce`, try updating it to 0.2. You may need a custom live ISO. You can also try adding the ``efi=noruntime`` option to GRUB's command line paramaters, or adding ``efivarfs /sys/firmware/efi/efivars efivarfs ro,remount,nofail 0 0`` to the end of your /etc/fstab file (although don't use both at the same time).
```

Comment:
What I said before was a bit vague, there can be other causes of crashes with symptoms like this, e.g. that nvme genctr that the t2 didn't like, messing with the usb cdc_ncm ethernet, amdgpu causing caterr. The specific crash when writing to nvram is linux jumping into some broken apple bootloader code, which has an invalid opcode and causes the panic, on the intel side.

With the nvram write crash, people can check if their kernel has the workaround for the broken efi runtime call.
`journalctl -k --grep="efi:"` must have "Apple Mac detected, using EFI v1.10 runtime services only".


> If you have `apple-bce`, try updating it to 0.2

apple-bce "0.1" and "0.2" isn't really a thing, just version numbers given to dkms. This means someone might have cloned it to apple-bce-0.2, but they might not have that repo updated to the most recent commit. In that case they should run git pull in /usr/src/apple-bce-0.2 and unbuild then install the modules (or update their kernel).